src/parameters/parameters_inits.jl

```diff
@@ -169,6 +169,7 @@ Base.@kwdef mutable struct FUSEparameters__tf{T} <: ParametersInit{T}
     ripple::Entry{T} =
         Entry{T}("-", "Fraction of toroidal field ripple evaluated at the outermost radius of the plasma chamber"; default=0.01, check=x -> @assert x > 0.0 "must be: ripple > 0.0")
     technology::Switch{Symbol} = Switch{Symbol}(FusionMaterials.supported_coil_techs(), "-", "TF coils technology")
+    nose_thickness::Entry{Float64} = Entry{Float64}("-", "Relative thickness of the TF nose, as a fraction of total TF radial build"; default=0.0)
```

Comment:
Why expressing nose thickness as a function of the total radial build? It's very hard to guess a reasonable initial value!

It would be a lot more intuitive to express it as a fraction of the TF radial build on the high-field-side... say 10%.

Maybe better calling it `nose_hfs_fraction`?

Reply:
"Total TF radial build" here means exactly this - the thickness of the TF coil on the high-field side. We can the parameter description to more clearly communicate this. But the actual code is already doing what you suggest.

Reply:
what threw me off was that you used `(hfsTF.end_radius-hfsTF.start_radius)` instead of `hfsTF.thickness`. This reinforces the point of using `nose_hfs_fraction`